intake/source/tests/test_utils.py

```diff
@@ -18,7 +18,8 @@
      'data/LT05_L1TP_042033_*_*_01_T1_sr_band*.tif'),
     ('{year}/{month}/{day}.csv', '*/*/*.csv'),
     ('data/**/*.csv', 'data/**/*.csv'),
-    ('data/{year:4}{month:02}{day:02}.csv', 'data/*.csv'),
+    ('data/{year:4}{month:02}{day:02}.csv', 'data/***.csv'),
```

Comment:
This is definitely not good - the output is not a legal glob string. The asterisks should combine.